compiler/renderer.nim

```diff
@@ -240,28 +240,6 @@ proc maxLineLength(s: string): int =
       inc(lineLen)
       inc(i)
 
-proc putRawStr(g: var TSrcGen, kind: TTokType, s: string) =
```

Comment:
deadcode but wondering if can be useful (with a deadcode comment) or safe to delete

Reply:
Readded it, since it might be useful for future renderer improvements.